README.md

```diff
@@ -1,473 +1,233 @@
 # CDC Trusted Intermediary
 
-This document provides instructions for setting up the environment, running the application, and performing various tasks such as compiling, testing, and contributing to the project.
+This document provides comprehensive instructions for setting up, running, and contributing to the project.
+
+---
+
+## Table of Contents
+
+1. [Requirements](#requirements)
```

Comment:
Is there a way to auto-generate this table? Otherwise we'll need to maintain it when we update add/remove sections

Reply:
I see what I did - on it. Will also investigate the auto generation

Reply:
One thing for auto generating ToC. If you use VS code, there's a plugin called 'Markdown - All In One' that has a ToC generator. I'm trying it out for our script read me.

Reply:
There is a built-in ToC in GitHub.  It is automatically generated based on the headings.  It is accessed by clicking on the button on the right hand side of the ReadMe.  See this image...

![image](https://github.com/user-attachments/assets/01cb03d2-e5e2-442e-ad5a-28ad8bf4dd00)